utils/print_qor.py

```diff
@@ -78,23 +77,32 @@ def main():
     )
     w = csv.DictWriter(sys.stdout, fields)
     w.writeheader()
+    rows = list()
     for root, dirs, files in os.walk(args.build_dir):
         if 'pack.log' in files:
             d = {}
 
-            d['path'] = root
-            d['pack time (sec)'] = scan_runtime(os.path.join(root, 'pack.log'))
+            path = root.split('/')
+            d['path'] = path[-2] + '/' + path[-1]
+            d['pack time (sec)'] = scan_runtime(
+                "Packing", os.path.join(root, 'pack.log')
+            )
             d['place time (sec)'] = scan_runtime(
-                os.path.join(root, 'place.log')
+                "Placement", os.path.join(root, 'place.log')
             )
             d['route time (sec)'] = scan_runtime(
-                os.path.join(root, 'route.log')
+                "Routing", os.path.join(root, 'route.log')
             )
             d['t_crit (ns)'], d['Fmax (MHz)'] = scan_critical(
```

Comment:
Request in meeting to add geomean CPD as well as CPD to table.

Reply:
Added both CPD geomean and its corresponding frequency